src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx

```diff
@@ -1828,7 +1828,7 @@
     <value>22</value>
   </data>
   <data name="TableLayoutPanel.Size" type="System.Drawing.Size, System.Drawing">
-    <value>2011, 917</value>
+    <value>1160, 900</value>
```

Comment:
This doesn't seem right to me.  Shouldn't we be using auto-sizing?  Something like setting `DockMode` to `Dock.Fill`?

Reply:
Agree. @etbyrd let's have a look at this one together.

Reply:
@Pilchie I believe this has been addressed now.